src/index.js

```diff
@@ -0,0 +1,170 @@
+"use strict";
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    tempValue: null,
+    landscape: null,
+    headerCityName: null,
+    cityNameInput: null,
+    currentTempButton: null,
+    temp: tempValue.textContent,
+    skySelect: null,
+    sky: null,
+    cityNameReset: null,
+    gardenContent: null,
+};
+
+const loadControls = () => {
+    for (const element in state) {
+        if (element !== 'temp') {
+            state[element] = document.getElementById(String(element));
+        }
+    }
+}
+loadControls()
+
+const changeTempColor = (temp) => {
+    let color;
+    let backgroundColor;
+    if (temp >= 80) {
+        color = 'red';
+        backgroundColor = "#a75051";
+    } else if (temp >= 70 && temp <= 79) {
+        color = 'orange';
+        backgroundColor = "#d3834d";
+    } else if (temp >= 60 && temp <= 69) {
+        color = 'yellow';
+        backgroundColor = "#c8a669";
+    } else if (temp >= 50 && temp <= 59) {
+        color = 'green';
+        backgroundColor = "#528478";
+    } else if (temp < 50) {
+        color = 'teal';
+        backgroundColor = "#4288c2";
+    }
+    state.tempValue.setAttribute('class', color);
+    document.body.style.background = backgroundColor;
+    cityNameReset.style.backgroundColor = backgroundColor;
+    currentTempButton.style.backgroundColor = backgroundColor;
+}
```

Comment:
```suggestion
};
```
since this is an assignment on line 29, we give it a semicolon at the end of the expression